src/app/pages/login/LoginPage.js

```diff
@@ -9,10 +9,14 @@ import {_login} from "../../../services/UserServices";
 
 class LoginPage extends Component {
     state = {
+        passwordShowed:true,
         auth: isAuthenticated(),
         loading: false,
         email: '',
-        password: ''
+        password: '',
+        errorMessage: '',
+        disabledButton: false,
```

Comment:
disabledButton
tên biến kiểu boolean thì nên đặt tên có is ở trước
thay vì đặt disabledButton -> isDisabledButton thì sẽ hiểu ngay là biến boolean
cơ mà tên biến chung chung quá, cũng hơi khó hiểu